lib/src/value/color.dart

```diff
@@ -81,6 +95,31 @@ class SassColor extends Value implements ext.SassColor {
         alpha = alpha == null ? 1 : fuzzyAssertRange(alpha, 0, 1, "alpha"),
         originalSpan = null;
 
+  factory SassColor.hwb(num hue, num whiteness, num blackness, [num alpha]) {
+    // From https://www.w3.org/TR/css-color-4/#hwb-to-rgb
+    var scaledHue = hue % 360 / 360;
+    var scaledWhiteness = fuzzyAssertRange(whiteness, 0, 100, "whiteness") / 100;
+    var scaledBlackness = fuzzyAssertRange(blackness, 0, 100, "blackness") / 100;
+
+    var sum = scaledWhiteness + scaledBlackness;
+    if (sum > 1) {
+      scaledWhiteness /= sum;
+      scaledBlackness /= sum;
+    }
+
+    var factor = 1 - scaledWhiteness - scaledBlackness;
+    int toRgb(num hue) {
+      var channel = _hueToRgb(0, 1, hue) * factor + scaledWhiteness;
+      return fuzzyRound(channel * 255);
+    }
+
+    return SassColor.rgb(
```

Comment:
It could be worth documenting why this returns an rgb color.

Reply:
Done.